I have no idea why the PR build fails on Travis, btw. Seems like the karma-browserify package won't install due to a checksum mismatch:

> ```
> npm ERR! shasum check failed for /tmp/npm-2071-cfd09410/registry.npmjs.org/karma-browserify/-/karma-browserify-4.1.1.tgz
> npm ERR! Expected: fe3e43f2eab1934ee08dc47a89575d52cb0bb77b
> npm ERR! Actual:   0a9fc7dc0e9aca0d4346067a2e61aee57afdadf1
> npm ERR! From:     https://registry.npmjs.org/karma-browserify/-/karma-browserify-4.1.1.tgz
> ```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -1,7 +1,6 @@
 var browserify = require('browserify');
 var gulp = require('gulp');
 var gutil = require('gulp-util');
-var ngAnnotate = require('browserify-ngannotate');
 var source = require('vinyl-source-stream');
 var watchify = require('watchify');
 
@@ -14,7 +13,7 @@ bundler.on('update', bundle);
 bundler.on('log', gutil.log);
 
 function bundle() {
-  return bundler.transform(ngAnnotate)
+  return bundler.transform('browserify-ngannotate', { 'single_quotes': true })
     .bundle()
     .on('error', gutil.log.bind(gutil, 'Bundle Error'))
     .pipe(source('hyper-content-for.js'))

